### PR TITLE
Update some kde5 applications to 18.04.3

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

File: srcpkgs/kcachegrind/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'kcachegrind'
 pkgname=kcachegrind
-version=18.04.2
+version=18.04.3
 revision=1
 build_style=cmake
 configure_args="-Wno-dev"
```
Comment (maxice8) [inline srcpkgs/kcachegrind/template]: is this still revelant ? it doesn't even look like a configuration switch for cmake

File: srcpkgs/kcron/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'kcron'
 pkgname=kcron
-version=18.04.2
+version=18.04.3
 revision=1
 build_style=cmake
 configure_args="-Wno-dev -DSYSCONF_INSTALL_DIR=/etc"
```
Comment (maxice8) [inline srcpkgs/kcron/template]: same here

File: srcpkgs/ksystemlog/template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'ksystemlog'
 pkgname=ksystemlog
-version=18.04.2
+version=18.04.3
 revision=1
 build_style=cmake
 configure_args="-Wno-dev -DSYSCONF_INSTALL_DIR=/etc"
```
Comment (maxice8) [inline srcpkgs/ksystemlog/template]: and here

File: srcpkgs/kcachegrind/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/kcachegrind/template]: Yes, that [cmake argument](https://cmake.org/cmake/help/latest/manual/cmake.1.html) can be safely dropped.

**Comment by maxice8** [comment]
kwalletmanager is failing cross

**Comment by Chocimier** [comment]
Unlike for kate, kauth problems can't be avoided, so marking nocross.

**Pull request merged**